Sources/LiveViewNative/Modifiers/Drawing and Graphics Modifiers/ContrastModifier.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  ContrastModifier.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/20/23.
+//
+
+import SwiftUI
+
+/// Sets the contrast and separation between similar colors in this view.
+///
+/// ```html
+/// <Circle modifiers={@native |> foreground_style(primary: {:color, :red}) |> contrast(amount: 0.5)} />
```

Comment:
I just want to make sure an integer won't blow up here: `contract(amount: 1)`

Reply:
@bcardarella  +1 on the sanity check since we don't have robust tests. I tested positive and negative integers as Elixir values and they worked as expected.